dnase/trackhub/update_tracks.bash

```diff
@@ -213,6 +213,7 @@ cp -R assets/${hub_type}/. ${hub_target}
 
 for i in "${genome_array[@]}"; do
     [ "${i}" = "${customGenomeAssembly}" ] && continue
+    [ "${i}" = "t2t" ] && continue
```

Comment:
why skipping these?

Reply:
The genomes native to the UCSC browser have simple, standard, two line stanzas in the genomes.txt file. They are written to genomes,txt here. The custom assemblies have more complicated, non-standard stanzas.  Those stanza lines are stored in the stub genomes.txt file in the assets/CEGS subdirectory, so we skip building them in this loop.

Reply:
add this info directly as a comment, also better to combine the two into a single if statement